pom.xml

```diff
@@ -61,16 +61,13 @@
     <dep.httpcore.version>5.2.1</dep.httpcore.version>
     <dep.impsort.version>1.8.0</dep.impsort.version>
     <dep.jackson.version>2.13.4</dep.jackson.version>
-    <dep.jakarta.activation-api.version>1.2.2</dep.jakarta.activation-api.version>
-    <dep.jakarta.servlet-api.version>4.0.3</dep.jakarta.servlet-api.version>
```

Comment:
This needs to get off of the 4.x line and up to 5.x+ to be truly compatible with jetty 11 and other jakarta libraries. It's my understanding that this not only needs to not be 4.x, but it can't be 6.x until Jetty 12 and the actual recommended version to use is the `org.eclipse.jetty.toolchain` library that we previously excluded in the dependency tree so I removed the exclusion on that.